client/js/lounge.js

```diff
@@ -494,73 +495,75 @@ $(function() {
 		userStyles: userStyles.text(),
 	}, JSON.parse(window.localStorage.getItem("settings")));
 
-	for (var i in options) {
-		if (i === "userStyles") {
-			if (!/[\?&]nocss/.test(window.location.search)) {
-				$(document.head).find("#user-specified-css").html(options[i]);
+	(function SettingsScope() {
```

Comment:
I wrapped them to reduce the scope (especially the I variable which was being shadowed in other functions)